overlord/snapstate/snapstate_update_test.go

```diff
@@ -6528,6 +6528,79 @@ func (s *validationSetsSuite) TestUpdateToRevisionSnapRequiredByValidationWronRe
 	c.Assert(err, ErrorMatches, `cannot update snap "some-snap" to revision 11 without --ignore-validation, revision 5 is required by validation sets: 16/foo/bar/2`)
 }
 
+func (s *validationSetsSuite) TestUpdateToRevisionSnapRequiredByValidationWrongRevisionIgnoreValidation(c *C) {
```

Comment:
nitpick: I'm a fan of long names myself :-), but there is a limit above which they become hard to understand, and I'm afraid this is one of them. In a case like this one I'd prefer to have a 2-3 lines comment that explains what this function does (if it's needed), or maybe several inline comments explaining the not obvious steps.

Reply:
Fair point, renamed with a comment. Thanks